<!DOCTYPE html><html><head><meta charSet="utf-8"/><title>Projects – Nextra</title><meta name="robots" content="index,follow"/><meta property="og:title" content="Projects – Nextra"/><meta name="theme-color" content="#111" media="(prefers-color-scheme: dark)"/><style>
        :root {
          --nextra-primary-hue: 212deg;
          --nextra-primary-saturation: 100%;
          --nextra-navbar-height: 4rem;
          --nextra-menu-height: 3.75rem;
          --nextra-banner-height: 2.5rem;
        }
        
        .dark {
          --nextra-primary-hue: 204deg;
          --nextra-primary-saturation: 100%;
        }
      </style><meta name="viewport" content="width=device-width, initial-scale=1.0"/><meta property="og:title" content="e]"/><meta property="og:description" content=""/><meta name="next-head-count" content="9"/><style class="_styletron_hydrate_">.ae{box-sizing:border-box}.af{border-radius:br1}.ag{justify-content:center}.ah{align-items:center}.ai{flex-direction:column}.aj{min-width:600px}.ak{color:#1d1d1e}.al{font-weight:500}.am{font-family:SF Pro Text}</style><link rel="preload" href="/_next/static/css/30240d624d8fdf24.css" as="style" crossorigin=""/><link rel="stylesheet" href="/_next/static/css/30240d624d8fdf24.css" crossorigin="" data-n-g=""/><link rel="preload" href="/_next/static/css/445c4173ffc2bf04.css" as="style" crossorigin=""/><link rel="stylesheet" href="/_next/static/css/445c4173ffc2bf04.css" crossorigin="" data-n-p=""/><noscript data-n-css=""></noscript><script defer="" crossorigin="" nomodule="" src="/_next/static/chunks/polyfills-c67a75d1b6f99dc8.js"></script><script src="/_next/static/chunks/webpack-6ef43a8d4a395f49.js" defer="" crossorigin=""></script><script src="/_next/static/chunks/framework-5429a50ba5373c56.js" defer="" crossorigin=""></script><script src="/_next/static/chunks/main-145f833db83c1e47.js" defer="" crossorigin=""></script><script src="/_next/static/chunks/pages/_app-3f42cb50bcc85940.js" defer="" crossorigin=""></script><script src="/_next/static/chunks/377-61a9680bfb71b9fa.js" defer="" crossorigin=""></script><script src="/_next/static/chunks/pages/projects-62aa541cb5e3f735.js" defer="" crossorigin=""></script><script src="/_next/static/eXifH6dA_s540iuYqD8px/_buildManifest.js" defer="" crossorigin=""></script><script src="/_next/static/eXifH6dA_s540iuYqD8px/_ssgManifest.js" defer="" crossorigin=""></script></head><body><div id="__next"><style>html { font-size: 16px } body { font-size: 14px; line-height: 24px; -webkit-font-smoothing: antialiased; font-family:"SF Pro Text", -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,"Helvetica Neue",Arial,"Noto Sans",sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol","Noto Color Emoji"; margin: 0; color: #000000; } p, h1, h2, h3, h4, h5, h6 { padding: 0; margin: 0; font-size: 14px; line-height: 24px;} input { border: none; font-family: inherit } a {
            margin: 0;
            padding: 0;
            color: #0284FE;
            cursor: pointer;
            text-decoration: none;
          } button { border: none } pre {
            margin: 0; padding: 1rem; font-size: 14px;
            line-height: 24px;
            font-family: monospace;
          } :focus {outline: none;}</style><div class="ae af ag ah ai aj ak al am "><script>!function(){try{var d=document.documentElement,c=d.classList;c.remove('light','dark');var e=localStorage.getItem('theme');if('system'===e||(!e&&true)){var t='(prefers-color-scheme: dark)',m=window.matchMedia(t);if(m.media!==t||m.matches){d.style.colorScheme = 'dark';c.add('dark')}else{d.style.colorScheme = 'light';c.add('light')}}else if(e){c.add(e|| '')}if(e==='light'||e==='dark')d.style.colorScheme=e}catch(e){}}()</script><div dir="ltr"><script>document.documentElement.setAttribute('dir','ltr')</script><div class="nextra-nav-container nx-sticky nx-top-0 nx-z-20 nx-w-full nx-bg-transparent print:nx-hidden"><div class="nextra-nav-container-blur nx-pointer-events-none nx-absolute nx-z-[-1] nx-h-full nx-w-full nx-bg-white dark:nx-bg-dark nx-shadow-[0_2px_4px_rgba(0,0,0,.02),0_1px_0_rgba(0,0,0,.06)] dark:nx-shadow-[0_-1px_0_rgba(255,255,255,.1)_inset] contrast-more:nx-shadow-[0_0_0_1px_#000] contrast-more:dark:nx-shadow-[0_0_0_1px_#fff]"></div><nav class="nx-mx-auto nx-flex nx-h-[var(--nextra-navbar-height)] nx-max-w-[90rem] nx-items-center nx-justify-end nx-gap-2 nx-pl-[max(env(safe-area-inset-left),1.5rem)] nx-pr-[max(env(safe-area-inset-right),1.5rem)]"><a class="nx-flex nx-items-center hover:nx-opacity-75 ltr:nx-mr-auto rtl:nx-ml-auto" href="/"><span></span></a><a href="https://github.com/edotau" target="_blank" rel="noreferrer" class="nx-p-2 nx-text-current"><svg width="24" height="24" fill="currentColor" viewBox="3 3 18 18"><title>GitHub</title><path d="M12 3C7.0275 3 3 7.12937 3 12.2276C3 16.3109 5.57625 19.7597 9.15374 20.9824C9.60374 21.0631 9.77249 20.7863 9.77249 20.5441C9.77249 20.3249 9.76125 19.5982 9.76125 18.8254C7.5 19.2522 6.915 18.2602 6.735 17.7412C6.63375 17.4759 6.19499 16.6569 5.8125 16.4378C5.4975 16.2647 5.0475 15.838 5.80124 15.8264C6.51 15.8149 7.01625 16.4954 7.18499 16.7723C7.99499 18.1679 9.28875 17.7758 9.80625 17.5335C9.885 16.9337 10.1212 16.53 10.38 16.2993C8.3775 16.0687 6.285 15.2728 6.285 11.7432C6.285 10.7397 6.63375 9.9092 7.20749 9.26326C7.1175 9.03257 6.8025 8.08674 7.2975 6.81794C7.2975 6.81794 8.05125 6.57571 9.77249 7.76377C10.4925 7.55615 11.2575 7.45234 12.0225 7.45234C12.7875 7.45234 13.5525 7.55615 14.2725 7.76377C15.9937 6.56418 16.7475 6.81794 16.7475 6.81794C17.2424 8.08674 16.9275 9.03257 16.8375 9.26326C17.4113 9.9092 17.76 10.7281 17.76 11.7432C17.76 15.2843 15.6563 16.0687 13.6537 16.2993C13.98 16.5877 14.2613 17.1414 14.2613 18.0065C14.2613 19.2407 14.25 20.2326 14.25 20.5441C14.25 20.7863 14.4188 21.0746 14.8688 20.9824C16.6554 20.364 18.2079 19.1866 19.3078 17.6162C20.4077 16.0457 20.9995 14.1611 21 12.2276C21 7.12937 16.9725 3 12 3Z"></path></svg><span class="nx-sr-only">GitHub</span><span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a><button type="button" aria-label="Menu" class="nextra-hamburger -nx-mr-2 nx-rounded nx-p-2 active:nx-bg-gray-400/20 md:nx-hidden"><svg fill="none" width="24" height="24" viewBox="0 0 24 24" stroke="currentColor" class=""><g><path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 6h16"></path></g><path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 12h16"></path><g><path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 18h16"></path></g></svg></button></nav></div><div class="nx-mx-auto nx-flex nx-max-w-[90rem]"><div class="motion-reduce:nx-transition-none [transition:background-color_1.5s_ease] nx-bg-transparent"></div><aside class="nextra-sidebar-container nx-flex nx-flex-col md:nx-top-16 md:nx-shrink-0 motion-reduce:nx-transform-none nx-transform-gpu nx-transition-all nx-ease-in-out print:nx-hidden md:nx-w-64 md:nx-sticky md:nx-self-start max-md:[transform:translate3d(0,-100%,0)]"><div class="nx-px-4 nx-pt-4 md:nx-hidden"></div><div class="nx-overflow-y-auto nx-overflow-x-hidden nx-p-4 nx-grow md:nx-h-[calc(100vh-var(--nextra-navbar-height)-var(--nextra-menu-height))] nextra-scrollbar"><div class="nx-transform-gpu nx-overflow-hidden nx-transition-all nx-ease-in-out motion-reduce:nx-transition-none"><div class="nx-transition-opacity nx-duration-500 nx-ease-in-out motion-reduce:nx-transition-none nx-opacity-100"><ul class="nx-flex nx-flex-col nx-gap-1 nextra-menu-desktop max-md:nx-hidden"><li class="nx-flex nx-flex-col nx-gap-1"><a class="nx-flex nx-rounded nx-px-2 nx-py-1.5 nx-text-sm nx-transition-colors [word-break:break-word] nx-cursor-pointer [-webkit-tap-highlight-color:transparent] [-webkit-touch-callout:none] contrast-more:nx-border nx-text-gray-500 hover:nx-bg-gray-100 hover:nx-text-gray-900 dark:nx-text-neutral-400 dark:hover:nx-bg-primary-100/5 dark:hover:nx-text-gray-50 contrast-more:nx-text-gray-900 contrast-more:dark:nx-text-gray-50 contrast-more:nx-border-transparent contrast-more:hover:nx-border-gray-900 contrast-more:dark:hover:nx-border-gray-50" href="/">Home</a></li><li class="nx-flex nx-flex-col nx-gap-1"><a class="nx-flex nx-rounded nx-px-2 nx-py-1.5 nx-text-sm nx-transition-colors [word-break:break-word] nx-cursor-pointer [-webkit-tap-highlight-color:transparent] [-webkit-touch-callout:none] contrast-more:nx-border nx-text-gray-500 hover:nx-bg-gray-100 hover:nx-text-gray-900 dark:nx-text-neutral-400 dark:hover:nx-bg-primary-100/5 dark:hover:nx-text-gray-50 contrast-more:nx-text-gray-900 contrast-more:dark:nx-text-gray-50 contrast-more:nx-border-transparent contrast-more:hover:nx-border-gray-900 contrast-more:dark:hover:nx-border-gray-50" href="/experience">Experience</a></li><li class="nx-flex nx-flex-col nx-gap-1 active"><a class="nx-flex nx-rounded nx-px-2 nx-py-1.5 nx-text-sm nx-transition-colors [word-break:break-word] nx-cursor-pointer [-webkit-tap-highlight-color:transparent] [-webkit-touch-callout:none] contrast-more:nx-border nx-bg-primary-100 nx-font-semibold nx-text-primary-800 dark:nx-bg-primary-400/10 dark:nx-text-primary-600 contrast-more:nx-border-primary-500 contrast-more:dark:nx-border-primary-500" href="/projects">Projects</a></li></ul></div></div></div></aside><nav class="nextra-toc nx-order-last nx-hidden nx-w-64 nx-shrink-0 xl:nx-block print:nx-hidden" aria-label="table of contents"></nav><div id="reach-skip-nav"></div><article class="nx-w-full nx-break-words nextra-content nx-flex nx-min-h-[calc(100vh-var(--nextra-navbar-height))] nx-min-w-0 nx-justify-center nx-pb-8 nx-pr-[calc(env(safe-area-inset-right)-1.5rem)]"><main class="nx-w-full nx-min-w-0 nx-max-w-6xl nx-px-6 nx-pt-4 md:nx-px-12"><h3 class="nx-font-semibold nx-tracking-tight nx-text-slate-900 dark:nx-text-slate-100 nx-mt-8 nx-text-2xl">Genomic Basis of Adaptive Evolution<a href="#genomic-basis-of-adaptive-evolution" id="genomic-basis-of-adaptive-evolution" class="subheading-anchor" aria-label="Permalink for this section"></a></h3>
<p class="nx-mt-6 nx-leading-7 first:nx-mt-0">Stickleback repeated adaptation to new environments provide a powerful mechanism to study the molecular genetics of evolutionary changes in wild populations. As we develop stickleback genomic sequencing technologies, we can start mapping genes that control evolutionary change to the genetic basis of adaptive evolution.</p>
<h3 class="nx-font-semibold nx-tracking-tight nx-text-slate-900 dark:nx-text-slate-100 nx-mt-8 nx-text-2xl">Marine Stickleback Genome<a href="#marine-stickleback-genome" id="marine-stickleback-genome" class="subheading-anchor" aria-label="Permalink for this section"></a></h3>
<p class="nx-mt-6 nx-leading-7 first:nx-mt-0">The newly minted marine stickleback genome was assembled into 22 chromosomes, 455MB in genome length with an N50 of 21,246,911 base pairs and nearly gapless with only 715 gaps remaining. The marine stickleback was from the Rabbit Slough, Alaska population was was sequenced by combining several long read sequencing technologies with comprehensive enhanced informative short read Illumina sequencing.</p>
<ul class="nx-mt-6 nx-list-disc first:nx-mt-0 ltr:nx-ml-6 rtl:nx-mr-6">
<li class="nx-my-2">Visualization: <a href="http://genome.ucsc.edu/cgi-bin/hgHubConnect?hgHub_do_redirect=on&amp;hgHubConnect.remakeTrackHub=on&amp;hgHub_do_firstDb=1&amp;hubUrl=http://trackhub.genome.duke.edu/lowelab/edotau/rabsTHREEspine/myHub/hub.txt" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">UCSC Genome Browser<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a></li>
<li class="nx-my-2">Chain Nets to other stickleback assemblies are available: <a href="https://edotau.github.io/chainNets" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">Here<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a></li>
<li class="nx-my-2">Stickleback Assembly: <a href="http://trackhub.genome.duke.edu/lowelab/edotau/rabsTHREEspine/rabsTHREEspine.fa" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">Download<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a></li>
<li class="nx-my-2">Raw Fastq Data Files: <a href="http://trackhub.genome.duke.edu/lowelab/edotau/rabsTHREEspine/raw_data/assembly/pacbio/" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">PacBio<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a> , <a href="http://trackhub.genome.duke.edu/lowelab/edotau/rabsTHREEspine/raw_data/assembly/10xGenomics" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">10x Genomics<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a> , <a href="http://trackhub.genome.duke.edu/lowelab/edotau/rabsTHREEspine/raw_data/assembly/hic" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">Hi-C<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a> , <a href="http://trackhub.genome.duke.edu/lowelab/edotau/rabsTHREEspine/raw_data/assembly/nanopore" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">Nanopore<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a></li>
</ul>
<h3 class="nx-font-semibold nx-tracking-tight nx-text-slate-900 dark:nx-text-slate-100 nx-mt-8 nx-text-2xl">Stickleback Genes and Transcripts<a href="#stickleback-genes-and-transcripts" id="stickleback-genes-and-transcripts" class="subheading-anchor" aria-label="Permalink for this section"></a></h3>
<ul class="nx-mt-6 nx-list-disc first:nx-mt-0 ltr:nx-ml-6 rtl:nx-mr-6">
<li class="nx-my-2">Ensembl Mapped: <a href="http://trackhub.genome.duke.edu/lowelab/edotau/rabsTHREEspine/myHub/genes/genePred/rabsTHREEspine.ensembl.mapped.gp" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">genePred<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a> , <a href="http://trackhub.genome.duke.edu/lowelab/edotau/rabsTHREEspine/myHub/genes/gtf/rabsTHREEspine.ensembl.mapped.gtf" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">GTF<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a> , <a href="http://trackhub.genome.duke.edu/lowelab/edotau/rabsTHREEspine/myHub/genes/gff/rabsTHREEspine.ensembl.mapped.gff" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">GFF3<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a></li>
<li class="nx-my-2">RNASEQ Genes: <a href="http://trackhub.genome.duke.edu/lowelab/edotau/rabsTHREEspine/myHub/genes/genePred/rna-seq.genes.mapped.ensembl.gp" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">genePred<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a> , <a href="http://trackhub.genome.duke.edu/lowelab/edotau/rabsTHREEspine/myHub/genes/gtf/rna-seq.genes.mapped.ensembl.gtf" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">GTF<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a> , <a href="http://trackhub.genome.duke.edu/lowelab/edotau/rabsTHREEspine/myHub/genes/gff/rna-seq.genes.mapped.ensembl.gff" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">GFF3<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a></li>
</ul>
<h3 class="nx-font-semibold nx-tracking-tight nx-text-slate-900 dark:nx-text-slate-100 nx-mt-8 nx-text-2xl">Genome-Wide Variation Between Freshwater and Marine Stickleback<a href="#genome-wide-variation-between-freshwater-and-marine-stickleback" id="genome-wide-variation-between-freshwater-and-marine-stickleback" class="subheading-anchor" aria-label="Permalink for this section"></a></h3>
<p class="nx-mt-6 nx-leading-7 first:nx-mt-0">Pairwise maine and freshwater stickleback genome alignments were produced in <a href="https://genome.ucsc.edu/goldenPath/help/axt.html" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">AXT<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a> format with <a href="http://www.bx.psu.edu/~rsharris/lastz/README.lastz-1.04.03.html" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">LASTZ<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a> and converted to <a href="https://genome.ucsc.edu/goldenPath/help/chain.html" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">CHAIN<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a> format with <a href="https://github.com/ENCODE-DCC/kentUtils/blob/master/src/hg/mouseStuff/axtChain/axtChain.c" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">axtChain<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a> . AXT and CHAIN alignments were generated in both directions. The set of alignments were computed using the marine stickleback as the target genome which used the freshwater stickleback as the query genome. However, the second set of alignments were computed in reverse, this time with freshwater genome as the target with the marine stickleback as its query genome and further processed using <a href="https://github.com/ENCODE-DCC/kentUtils/blob/master/src/hg/mouseStuff/chainSwap/chainSwap.c" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">chainSwap<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a> . LASTZ and axtChain were programmed into a wrapping <a href="https://raw.githubusercontent.com/edotau/sticklebackCipher/master/lastz/lastz.sh" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">script<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a> . All chains were merged and sorted with <a href="https://github.com/ENCODE-DCC/kentUtils/blob/master/src/hg/mouseStuff/chainMergeSort/chainMergeSort.c" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">chainMergeSort<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a> . Finally the best alignments from chains are selected with <a href="https://github.com/ENCODE-DCC/kentUtils/blob/master/src/hg/mouseStuff/chainNet/chainNet.c" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">chainNet<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a> using a red-black trees algorithm to keep track of areas of a chromosome are already covered until there are no bases left and distinguish duplicated from non-duplicated regions. The resulting file is hierarchical collection of chains, with the highest-scoring non-overlapping chains on top, and gaps filled by possible lower-scoring chains called a NET. Nets are single-coverage target genomes, but not for query genomes unless it has been filtered to be single-coverage on both target and query. We generated reciprocal-best netted chains our pairwise netted chains by writing a wrapping <a href="https://raw.githubusercontent.com/edotau/sticklebackCipher/master/lastz/reciprocalBestChainNets.sh" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">bash script<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a> . All programs to process chains and nets are open sourced UCSC Kent Utilities developed to examines genomic duplications, deletions, and rearrangements of the first whole-genome comparisons between human and mouse <a href="https://www.pnas.org/content/pnas/100/20/11484.full.pdf" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">J Kent 2003<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a>.</p>
<h4 class="nx-font-semibold nx-tracking-tight nx-text-slate-900 dark:nx-text-slate-100 nx-mt-8 nx-text-xl">Marine Freshwater Hybrid Crosses<a href="#marine-freshwater-hybrid-crosses" id="marine-freshwater-hybrid-crosses" class="subheading-anchor" aria-label="Permalink for this section"></a></h4>
<ul class="nx-mt-6 nx-list-disc first:nx-mt-0 ltr:nx-ml-6 rtl:nx-mr-6">
<li class="nx-my-2">GATK: <a href="http://trackhub.genome.duke.edu/lowelab/edotau/rabsTHREEspine/myHub/vcfs/snps" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">SNPS<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a>, <a href="http://trackhub.genome.duke.edu/lowelab/edotau/rabsTHREEspine/myHub/vcfs/indels" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">INDELS<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a></li>
<li class="nx-my-2">Two different F1 hybrid (Freshwater x Marine) <a href="https://en.wikipedia.org/wiki/Three-spined_stickleback" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">stickleback<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a> fish were crossed from <a href="https://www.google.com/maps/place/Matadero+Creek/@37.4241044,-122.1510844,14z/data=!3m1!4b1!4m5!3m4!1s0x808fbaf41f10715b:0xc780fcc2d1157053!8m2!3d37.4241062!4d-122.1335748" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">Matadero Creek<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a> with <a href="https://www.google.com/maps/place/Little+Campbell+River/@49.0339164,-122.7053782,13z/data=!3m1!4b1!4m5!3m4!1s0x5485c5e345db5175:0x48382b94481b427a!8m2!3d49.0339216!4d-122.6703588" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">Little Campbell<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a> and <a href="https://www.google.com/maps/place/Rabbit+Slough/@61.5398359,-149.3153327,14z/data=!4m5!3m4!1s0x56c8e14ff29fb997:0x4542770c26b5618a!8m2!3d61.5333333!4d-149.2666666" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">Rabbit Slough<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a> with <a href="https://www.google.com/maps/place/Matanuska+Lake/@61.555098,-149.2414204,15z/data=!3m1!4b1!4m5!3m4!1s0x56c8e17081d30ab5:0x911087de19454c0b!8m2!3d61.5550708!4d-149.231791" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">Lake Matanuska<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a>. A Summary Table of populations sampled and libraries sequenced to investigate each allele are described in panel B. We generated <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4374986/" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">ATAC-Seq<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a> , <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2949280/" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">RNA-Seq<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a>, and <a href="https://www.internationalgenome.org/sites/1000genomes.org/files/docs/nature09534.pdf" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">WGS<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a> libraries to screen for allele specific enhancers near regions of increased chromatin accessibility that could perhaps be regulating differential gene expression. We perform a <a href="https://en.wikipedia.org/wiki/Fisher%27s_exact_test" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">Fisher&#x27;s Exact Test<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a> genome-wide to determine statistical significance if heterogygous SNPs sites are differentially increased or decreased in chromatin accessibility are enhancers or regulatory elements nearby differentially expressed genes.</li>
</ul>
<h3 class="nx-font-semibold nx-tracking-tight nx-text-slate-900 dark:nx-text-slate-100 nx-mt-8 nx-text-2xl">Differential Chromatin Accessibility and Gene Expression<a href="#differential-chromatin-accessibility-and-gene-expression" id="differential-chromatin-accessibility-and-gene-expression" class="subheading-anchor" aria-label="Permalink for this section"></a></h3>
<ul class="nx-mt-6 nx-list-disc first:nx-mt-0 ltr:nx-ml-6 rtl:nx-mr-6">
<li class="nx-my-2">Results: <a href="http://trackhub.genome.duke.edu/lowelab/edotau/rabsTHREEspine/myHub/beds/differentialChromatin/" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">Download<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a>
<img src="/pictures/pilotF1Hybrid.svg" alt="Photo" height="800" width="1200" class="next-image"/>
</li>
</ul>
<h3 class="nx-font-semibold nx-tracking-tight nx-text-slate-900 dark:nx-text-slate-100 nx-mt-8 nx-text-2xl">Differential Chromatin Accessibility and Gene Expression<a href="#differential-chromatin-accessibility-and-gene-expression-1" id="differential-chromatin-accessibility-and-gene-expression-1" class="subheading-anchor" aria-label="Permalink for this section"></a></h3>
<ul class="nx-mt-6 nx-list-disc first:nx-mt-0 ltr:nx-ml-6 rtl:nx-mr-6">
<li class="nx-my-2">
<p class="nx-mt-6 nx-leading-7 first:nx-mt-0">Results: <a href="http://trackhub.genome.duke.edu/lowelab/edotau/rabsTHREEspine/myHub/beds/differentialChromatin/" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">Download<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a></p>
<img src="/pictures/chr04_30595050_30815018.svg" alt="Photo" height="400" width="900" class="next-image"/>
<p class="nx-mt-6 nx-leading-7 first:nx-mt-0">After genotyping 18 stickleback genomes and 4 transcriptomes, 9,468,248 SNPs genome wide were detcted. Testing each heterogygous SNPs site pinpointed to this PLXNA4 gene located near the end of chromosome 4 spanning a little more than 200 kb.</p>
<p class="nx-mt-6 nx-leading-7 first:nx-mt-0">In the F1 hybrid WGS data is heterozygous for both SNPs. However, in the atac-seq cross, the genotype is homozygous suggesting one of the alleles is differentially increased in chromatin accessibility, while the other allele is completely closed. What is even more interesting is nearby in the rna-seq data shows gene expression is clearly down-regulated in one allele and increased gene expression with the other allele.</p>
<p class="nx-mt-6 nx-leading-7 first:nx-mt-0">What&#x27;s PLXNA4? <a href="https://www.omim.org/entry/604280" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">PLXNA4<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a> is a receptor for proteins SEMA3A and SEMA6, which mediate the effects of multiple semaphorins, including controlling diverse aspects of the nervous system development and plasticity, ranging from axon guidance and neuron migration to synaptic organization <a href="https://science.sciencemag.org/content/342/6158/1241974" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">Sun et al., 2013<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a>. PLXNA4 has recently been identified in genome wide association studies (GWAS), as a novel genetic player associated with Alzheimer&#x27;s disease <a href="https://www.frontiersin.org/articles/10.3389/fnins.2018.00946/full" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">Han et al., 2018<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a>.</p>
<h4 class="nx-font-semibold nx-tracking-tight nx-text-slate-900 dark:nx-text-slate-100 nx-mt-8 nx-text-xl">Raw Data<a href="#raw-data" id="raw-data" class="subheading-anchor" aria-label="Permalink for this section"></a></h4>
<ul class="nx-mt-6 nx-list-disc first:nx-mt-0 ltr:nx-ml-6 rtl:nx-mr-6">
<li class="nx-my-2">ATAC-Seq: <a href="http://trackhub.genome.duke.edu/lowelab/edotau/rabsTHREEspine/raw_data/atacseq/" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">fastqs<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a>, <a href="http://trackhub.genome.duke.edu/lowelab/edotau/rabsTHREEspine/myHub/bams/atacseq/" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">bams<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a>, <a href="http://trackhub.genome.duke.edu/lowelab/edotau/rabsTHREEspine/myHub/bams/atacseq/peakCalls" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">peaks<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a></li>
<li class="nx-my-2">RNA-Seq: <a href="http://trackhub.genome.duke.edu/lowelab/edotau/rabsTHREEspine/raw_data/atacseq/" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">fastqs<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a>, <a href="http://trackhub.genome.duke.edu/lowelab/edotau/rabsTHREEspine/myHub/bams/rnaseq" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">bams<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a> , <a href="http://trackhub.genome.duke.edu/lowelab/edotau/rabsTHREEspine/myHub/genePred" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">genePred<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a>, <a href="http://trackhub.genome.duke.edu/lowelab/edotau/rabsTHREEspine/myHub/beds/rna_seq/" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">gene enrichments<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a></li>
<li class="nx-my-2">Genome-Seq: <a href="http://trackhub.genome.duke.edu/lowelab/edotau/rabsTHREEspine/raw_data/genomeseq" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">fastqs<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a>, <a href="http://trackhub.genome.duke.edu/lowelab/edotau/rabsTHREEspine/myHub/bams/genomeseq" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">bams<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a></li>
</ul>
<h4 class="nx-font-semibold nx-tracking-tight nx-text-slate-900 dark:nx-text-slate-100 nx-mt-8 nx-text-xl">Chromosome Spanning Assemblies<a href="#chromosome-spanning-assemblies" id="chromosome-spanning-assemblies" class="subheading-anchor" aria-label="Permalink for this section"></a></h4>
<p class="nx-mt-6 nx-leading-7 first:nx-mt-0">Hi-C libraries are commonly used today for assembling draft genomes largely because these libraries can provide insight into 3-D chomosome structures and investigate chromosome interactions in close proximity. A freshwater three-spine stickleback Hi-C library previously provided a noticeable improvement to the Gasterosteus aculeatus freshwater assembly <a href="https://academic.oup.com/jhered/article/108/6/693/3957968" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">CL Peichel, 2017<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a>
However, Karyotype characterization in sticklebacks have previously reported to have a high divergence chromosomal morphology <a href="https://www.nature.com/articles/nature08441" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">Kitano 2009<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a> , <a href="https://pubmed.ncbi.nlm.nih.gov/19229325/" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">Ross 2009<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a> When we mapped previously generated freshwater Hi-C data to both the freshwater assembly and the marine stickleback genome revealed identical heat map signatures. Similarly when we mapped marine H-C data against the freshwater and marine genomes respectfully, produced the same Hi-C heat map profiles suggesting both assemblies are high contiguity chromosome level assemblies, but no large detectable chromosomal rearrangements or translocations.</p>
<img src="/pictures/hic-plot.svg" alt="Photo" height="900" width="900" class="next-image"/>
</li>
</ul>
<h4 class="nx-font-semibold nx-tracking-tight nx-text-slate-900 dark:nx-text-slate-100 nx-mt-8 nx-text-xl">Marine Freshwater Chomosomal Inversions<a href="#marine-freshwater-chomosomal-inversions" id="marine-freshwater-chomosomal-inversions" class="subheading-anchor" aria-label="Permalink for this section"></a></h4>
<p class="nx-mt-6 nx-leading-7 first:nx-mt-0">Performing marine to freshwater genome alignments and comparisons reveal consistent findings with previous work which identified 3 marine freshwater chromosome inversions located on chromosomes 1, 11, and 21 <a href="https://www.nature.com/articles/nature10944.pdf" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">Jones et al. 2012<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a>. On the marine stickleback genome coordinates, the inversions span roughly from 26.630.694 to 27,081,942 on chromosome one, 6,311,704 to 6,740,674 on chromosome eleven, and 9,570,312 to 11,281,238 on chromosome twenty-one.</p>
<img src="/pictures/inversions.svg" alt="Photo" height="300" width="700" class="next-image"/>
<h4 class="nx-font-semibold nx-tracking-tight nx-text-slate-900 dark:nx-text-slate-100 nx-mt-8 nx-text-xl">Presence Absence<a href="#presence-absence" id="presence-absence" class="subheading-anchor" aria-label="Permalink for this section"></a></h4>
<p class="nx-mt-6 nx-leading-7 first:nx-mt-0">PITX1 contains multiple regulatory switches that allow for transcription of that gene in multiple tissues.</p>
<img src="/pictures/pitx1_PA.svg" alt="Photo" height="400" width="900" class="next-image"/>
<h4 class="nx-font-semibold nx-tracking-tight nx-text-slate-900 dark:nx-text-slate-100 nx-mt-8 nx-text-xl">Genomic Software Projects<a href="#genomic-software-projects" id="genomic-software-projects" class="subheading-anchor" aria-label="Permalink for this section"></a></h4>
<ul class="nx-mt-6 nx-list-disc first:nx-mt-0 ltr:nx-ml-6 rtl:nx-mr-6">
<li class="nx-my-2">github.com/edotau/<a href="https://github.com/edotau/goFish" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">goFish<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a></li>
<li class="nx-my-2">Slurm Compute Cluster: <a href="https://github.com/edotau/sticklebackCipher" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">sticklebackCipher<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a></li>
</ul>
<h4 class="nx-font-semibold nx-tracking-tight nx-text-slate-900 dark:nx-text-slate-100 nx-mt-8 nx-text-xl">Lowe Lab Software<a href="#lowe-lab-software" id="lowe-lab-software" class="subheading-anchor" aria-label="Permalink for this section"></a></h4>
<ul class="nx-mt-6 nx-list-disc first:nx-mt-0 ltr:nx-ml-6 rtl:nx-mr-6">
<li class="nx-my-2">Website: <a href="http://www.vertgenlab.org/" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">vertgenlab<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a></li>
<li class="nx-my-2">github.com/ <a href="https://github.com/vertgenlab/gonomics" target="_blank" rel="noreferrer" class="nx-text-primary-600 nx-underline nx-decoration-from-font [text-underline-position:from-font]">gonomics<span class="nx-sr-only nx-select-none"> (opens in a new tab)</span></a></li>
</ul><div class="nx-mt-16"></div><div class="nx-mb-8 nx-flex nx-items-center nx-border-t nx-pt-8 dark:nx-border-neutral-800 contrast-more:nx-border-neutral-400 dark:contrast-more:nx-border-neutral-400 print:nx-hidden"><a title="Experience" class="nx-flex nx-max-w-[50%] nx-items-center nx-gap-1 nx-py-4 nx-text-base nx-font-medium nx-text-gray-600 nx-transition-colors [word-break:break-word] hover:nx-text-primary-600 dark:nx-text-gray-300 md:nx-text-lg ltr:nx-pr-4 rtl:nx-pl-4" href="/experience"><svg fill="none" viewBox="0 0 24 24" stroke="currentColor" class="nx-inline nx-h-5 nx-shrink-0 ltr:nx-rotate-180"><path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M9 5l7 7-7 7"></path></svg>Experience</a></div></main></article></div><footer class="nx-bg-gray-100 nx-pb-[env(safe-area-inset-bottom)] dark:nx-bg-neutral-900 print:nx-bg-transparent"><div class="nx-mx-auto nx-flex nx-max-w-[90rem] nx-gap-2 nx-py-2 nx-px-4 nx-hidden"></div><hr class="dark:nx-border-neutral-800"/><div class="nx-mx-auto nx-flex nx-max-w-[90rem] nx-justify-center nx-py-12 nx-text-gray-600 dark:nx-text-gray-400 md:nx-justify-start nx-pl-[max(env(safe-area-inset-left),1.5rem)] nx-pr-[max(env(safe-area-inset-right),1.5rem)]"><span>edotau et. al. © <!-- -->2024</span></div></footer></div></div></div><script id="__NEXT_DATA__" type="application/json" crossorigin="">{"props":{"pageProps":{}},"page":"/projects","query":{},"buildId":"eXifH6dA_s540iuYqD8px","nextExport":true,"autoExport":true,"isFallback":false,"scriptLoader":[]}</script></body></html>